### DISCUSSION TITLE
Am I keeping up with the checkout new branches feature?

### DISCUSSION THREAD

**Discussion by codeDude64** [NeogitOrg/neogit#326] [Q&A] [answered]
I guess  several weeks ago, when I updated my nvim plugins I realized the feature "checkout new branch" now shows a list of branches instead of just listen a input to introduce the new branch name.
I don't know if it is a bug, or is a new feature.

What is this list of branches for?
![neogit_example](https://user-images.githubusercontent.com/35637860/182005476-1e9602ec-b648-4297-80e6-6a2c1cbb609c.gif)

**Comment by codeDude64** [comment, marked as answer]
Today I got time to check the code and It looks like is intentionally
![image](https://user-images.githubusercontent.com/35637860/189512300-ade60707-c38b-430e-a435-950c22cc96c1.png)

Honestly, I don't get Why the `selected_branch` shouldn't be the `current_branch` I was going to create a PR but I guess the current code have a good reason

**Reply by kessejones** [reply]
Hello @codeDude09 
This issue has already been resolved in #309
